### PR TITLE
Improve wording and translation markup for commandline args

### PR DESCRIPTION
**Pull request by cas--**

### CODE REVIEW

File: deluge/ui/web/web.py [hunk 1]
```diff
@@ -36,33 +36,33 @@ def __init__(self, *args, **kwargs):
...
         if not windows_check():
             group.add_argument("-U", "--user", metavar="<user>", action="store", default=None,
-                               help="User to switch to. Only use it when starting as root")
+                               help=_("User to switch to. Only use it when starting as root"))
             group.add_argument("-g", "--group", metavar="<group>", action="store", default=None,
-                               help="Group to switch to. Only use it when starting as root")
+                               help=_("Group to switch to. Only use it when starting as root"))
         group.add_argument("-i", "--interface", metavar="<interface>", action="store", default=None,
-                           help="Binds the webserver to a specific IP address")
+                           help=_("Binds the webserver to a specific IP address"))
```
Comment (bendikro) [inline deluge/ui/web/web.py]: I think we should adhere to consistency in the option description and maybe standardize on a commanding the program. e.g Change this line to `Bind the webserver to a specific IP address`

Comment (cas--) [inline deluge/ui/web/web.py]: I've modified it to match the daemon text

**Comment by cas--** [comment]
Should add a description for each of the grouping, where helpful...

**Comment by cas--** [comment]
Also do we really need that OpenSSL check in web entry...

File: deluge/ui/baseargparser.py [hunk 2]
```diff
@@ -139,12 +139,58 @@ def parse_args(self, *args):
...
+        """Adds a grouping of common process args to control a daemon to the parser"""
+
+        self.process_group = True
+        self.group = self.add_argument_group(_("Process Control Options"))
+        self.group.add_argument("-P", "--pidfile", metavar="<pidfile>", action="store",
+                                help=_("Pidfile to store the process id"))
+        if not common.windows_check():
+            self.group.add_argument("-d", "--do-not-daemonize", dest="donotdaemonize", action="store_true",
+                                    help=_("Do not daemonize (fork) this process"))
+            self.group.add_argument("-f", "--fork", dest="donotdaemonize", action="store_false",
```
Comment (bendikro) [inline deluge/ui/baseargparser.py]: This actually adds the `--fork` option to the daemon, which never had that option. Not really a big deal tho...

Comment (cas--) [inline deluge/ui/baseargparser.py]: It pained me to leave it there :P

**Pull request merged**